src/coreComponents/mesh/generators/VTKMeshGenerator.cpp

```diff
@@ -150,18 +181,32 @@ loadMesh( Path const & filePath )
   return loadedMesh;
 }
 
+vtkNew< vtkCellArray > GetCellArray( vtkDataSet & mesh ) // replaces GetCells() that exist only in vtkUnstructuredGrid
+{
+  vtkNew< vtkCellArray > cells;
+  if( mesh.IsA( "vtkStructuredGrid" ) || mesh.IsA( "vtkUnstructuredGrid" ))
+  {
+    int numCell = mesh.GetNumberOfCells();
+    for( int c = 0; c < numCell; c++ )
+    {
+      cells->InsertNextCell( mesh.GetCell( c ));
+    }
+  }
+  return cells;
+}
+
 template< typename INDEX_TYPE, typename POLICY >
 ArrayOfArrays< INDEX_TYPE, INDEX_TYPE >
-buildElemToNodesImpl( vtkUnstructuredGrid & mesh )
+buildElemToNodesImpl( vtkDataSet & mesh,
+                      const vtkNew< vtkCellArray > & cells )
```

Comment:
We use east-const convention throughout the code, would you mind making sure the changes in this file are consistent with it?
```suggestion
                      vtkNew< vtkCellArray > const & cells )
```